### PR TITLE
Add focus outline when focusing with keyboard

### PR DESCRIPTION
**Pull request by pajong**
### This PR will...
Add focus outline when focusing with keyboard

#### Addresses Issue(s):

JW8-2330

### CODE REVIEW

**Comment by johnBartos** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  🛠 There are modified src files, but no test changes. Add tests if you're able to.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  🗿 Set a milestone. It should be the ticket's fix version in JIRA.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by karimMourra** [comment]
@pajong, i took for it a spin; it looks great but the cast icon does not show the focus ring
http://player-pr-test-jenkins.longtailvideo.com/builds/18834/archive/test/squash/bootstrap.html?primary=html5&edition=ads&feature=player_ui/player_ui_with_the_kitchen_sink&suite=Player%20UI%20and%20Customization

File: src/css/controls/imports/icons.less [hunk 1]
```diff
@@ -7,6 +7,10 @@
 .jw-icon {
     .square(44px);
     background-color: transparent;
+
+    &:not(.jw-no-focus)&:focus svg {
```
Comment (karimMourra) [inline src/css/controls/imports/icons.less]: this outputs to `.jw-icon:not(.jw-no-focus).jw-icon:focus svg`; when do we have this ?

File: src/css/controls/imports/settings-menu.less [hunk 2]
```diff
@@ -107,7 +107,10 @@
 
     &:focus {
         font-weight: bold;
-        outline: none;
+
+        &&:not(.jw-no-focus) {
```
Comment (karimMourra) [inline src/css/controls/imports/settings-menu.less]: outputs to `.jw-settings-content-item:focus.jw-settings-content-item:focus:not(.jw-no-focus) {` - should this be
`.jw-settings-content-item:focus:not(.jw-no-focus)` instead ?

File: src/css/controls/imports/slider.less [hunk 3]
```diff
@@ -118,6 +118,10 @@
     z-index: 1;
     outline: none;
 
+    &&:not(.jw-no-focus)&:focus .jw-slider-container .jw-rail {
```
Comment (karimMourra) [inline src/css/controls/imports/slider.less]: outputs `.jw-slider-time.jw-slider-time:not(.jw-no-focus).jw-slider-time:focus .jw-slider-container .jw-rail` - should it be shortened to `.jw-slider-time:focus:not(.jw-no-focus) .jw-slider-container .jw-rail` ?

**Pull request merged**